pipeline_games/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY pipeline.sh .
```

Comment:
I like the order here, we're most likely to change the python scripts so it makes sense to copy them last